packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1716,11 +1737,11 @@ class GridMetricCalculator {
   /**
    * Sets the width for the specified row
    * @param row The row model index to set
-   * @param size The size to set it to, or null to reset the row size
+   * @param size The size to set it to, or undefined to reset the row size
    */
   setRowHeight(row: ModelIndex, size?: number): void {
     const userRowHeights = new Map(this.userRowHeights);
-    if (size != null) {
+    if (size !== undefined) {
```

Comment:
Same as above

Reply:
Yea I'm just going to force calling into `resetRowHeight` instead, and require a number here.

All places that call `setRowHeight`/`setColumnWidth` pass a value